### PR TITLE
Update to Confex 3.2

### PR DESCRIPTION
**Pull request by minijackson**
Just what the title says ;-)

### CODE REVIEW

**Comment by AndrewDryga** [comment]
Hello, @minijackson! This is your first Pull Request that will be reviewed by [Ebert](https://ebertapp.io/), an automatic Code Review service. It will leave comments on this diff with potential issues and style violations found in the code as you push new commits. You can also see all the issues found on this Pull Request on its [review page](https://ebertapp.io/github/Nebo15/ecto_mnesia/pulls/48). Please check our [documentation](https://docs.ebertapp.io) for more information.

File: lib/ecto_mnesia/storage.ex [hunk 1]
```diff
@@ -69,7 +69,7 @@ defmodule EctoMnesia.Storage do
   end
 
   def conf(config \\ []) do
-    Confex.process_env([
+    Confex.Resolver.resolve!([
```
Comment (AndrewDryga) [inline lib/ecto_mnesia/storage.ex:72]: Nested modules could be aliased at the top of the invoking module.

**Comment by AndrewDryga** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 10 possible new issues (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/Nebo15/ecto_mnesia/pulls/48.

**Comment by AndrewDryga** [comment]
@minijackson thanks 👍 . 

This change was already on my local branch, I've just started to work on updating deps, including support for latest Ecto release :)

**Comment by minijackson** [comment]
Great! I'll be happy to see that release

**Pull request merged**